src/nnfusion/engine/pass/graph/kernel_tuning.cpp

```diff
@@ -17,30 +17,16 @@ using namespace nnfusion::kernels;
 using namespace nnfusion::pass::graph;
 
 DEFINE_int64(fkernel_tuning_steps, 0, "Enable automatic kernel tuning for maximum N steps.");
+DEFINE_string(fkernel_tuning_config,
+              "",
+              "List of op types that use specified tuning stpes, e.g., \"Conv:200,Matmul:300\"");
```

Comment:
shall we use json format like [-ftraining_optimizer](https://github.com/microsoft/nnfusion/blob/master/src/nnfusion/engine/pass/graph/autodiff_pass.cpp#L30)?